glibc.spec

```diff
@@ -12,26 +12,27 @@ URL: http://www.gnu.org/software/libc/
 Source0: glibc-2.30.tar.xz
 Source1: build-locale-archive.c
 
-Patch1: glibc-arm-alignment-fix.patch
-Patch2: glibc-2.25-arm-runfast.patch
-Patch4: glibc-2.25-elf-rtld.diff
-Patch5: glibc-2.27-ldso-rpath-prefix-option.diff
-Patch6: glibc-2.25-nsswitchconf-location.diff
-Patch7: glibc-2.25-nscd-socket-location.diff
-Patch8: glibc-2.25-ldso-nodefaultdirs-option.diff
-Patch9: glibc-2.14-locarchive-fedora.patch
-Patch10: eglibc-2.15-fix-neon-libdl.patch
-Patch11: eglibc-2.19-shlib-make.patch
-Patch12: glibc-2.27-bits.patch
-Patch13: 0001-Revert-elf-Refuse-to-dlopen-PIE-objects-BZ-24323.patch
-Patch14: 0002-arm-CVE-2020-6096-fix-memcpy-and-memmove-for-negativ.patch
-Patch15: 0003-arm-CVE-2020-6096-Fix-multiarch-memcpy-for-negative-.patch
-Patch16: 0001-Fix-array-bounds-violation-in-regex-matcher-bug-2514.patch
-Patch17: 0001-posix-Sync-regex-with-gnulib.patch
-Patch18: glibc-CVE-2020-27618.patch
-Patch19: glibc-CVE-2021-33574-mq_notify-use-after-free-pthread-attribute.patch
-Patch20: glibc-CVE-2021-33574-backport-fix-use-pthread-attr-copy.patch
-
+Patch1: 0001-Fix-libc6-alignment-error-in-lib-ld-linux.so.3-on-ar.patch
+Patch2: 0002-ARM-default-to-FPU-RunFast-mode.patch
+Patch3: 0003-Fix-locale-archives-fixing-MER-295.patch
+Patch4: 0004-Fix-crash-when-linking-with-libdl-on-arm-with-NEON.patch
+Patch5: 0005-Fix-shlib.lds-generation.patch
+Patch6: 0006-scratchbox2-Allow-specifying-of-nscd-socket-location.patch
+Patch7: 0007-scratchbox2-Allow-specifying-of-nsswitch.conf-path.patch
+Patch8: 0008-scratchbox2-Add-options-to-not-set-any-default-dirs-.patch
+Patch9: 0009-scratchbox2-Add-ld.so-argv0-STRING-to-use-STRING-as-.patch
+Patch10: 0010-scratchbox2-Add-new-option-rpath-prefix-to-ld.so.patch
+Patch11: 0011-Define-bits-endian.h-instead-of-generating-it.patch
+Patch12: 0012-Revert-elf-Refuse-to-dlopen-PIE-objects-BZ-24323.patch
+Patch13: 0013-arm-CVE-2020-6096-fix-memcpy-and-memmove-for-negativ.patch
+Patch14: 0014-arm-CVE-2020-6096-Fix-multiarch-memcpy-for-negative-.patch
+Patch15: 0015-Fix-array-bounds-violation-in-regex-matcher-bug-2514.patch
+Patch16: 0016-posix-Sync-regex-with-gnulib.patch
+Patch17: 0017-iconv-Accept-redundant-shift-sequences-in-IBM1364-BZ.patch
+Patch18: 0018-Use-__pthread_attr_copy-in-mq_notify-bug-27896.patch
+Patch19: 0019-Fix-use-of-__pthread_attr_copy-in-mq_notify-bug-2789.patch
```

Comment:
There is something nice about seeing the CVE mentioned in the filename for a quick glance.